modules/olm-po-techpreview.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-po.adoc
+
+:_content-type: CONCEPT
+[id="olm-po-techpreview_{context}"]
+= Technology Preview restrictions for platform Operators
+
+During the Technology Preview release of the platform Operators feature in {product-title} 4.12, the following restrictions determine whether an Operator can be installed through the platform Operators mechanism:
+
+* Kubernetes manifests must be packaged using the Operator Lifecycle Manager (OLM) `registry+v1` bundle format
+* Cannot declare package or GVK dependencies
```

Comment:
Suggestion: add "You" to this bullet and the next and then all of these could be sentence items with a period. This would help the list to be more parallel in structure.

Reply:
Question: Should GVK be spelled out at first use?

Reply:
Indeed, I'm even the one who added it to the term glossary in the first place! 😅 https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#groupversionkind-gvk